src/Index.php

```diff
@@ -666,6 +673,9 @@ public function save(EntityInterface $entity, $options = [])
 
         $doc = new ElasticaDocument($id, $data);
         $doc->setAutoPopulate(true);
+        if ($options['routing']) {
```

Comment:
Can the routing be 0?

Reply:
I suppose that technically it can be, will change.